### DISCUSSION TITLE
Ability to set a fixed window width

### DISCUSSION THREAD

**Discussion by x9136** [Chatterino/chatterino2#5249] [Ideas] [answered]
I suggest adding a setting in which you can specify the desired window width and lock it to avoid accidental changes. This is necessary so that the chat always looks the way you need relative to the rest of the windows on the desktop or relative to the length of the chat lines convenient for you.

A simplified option is to add a checkbox so that the chat always opens in the width that allows the chat lines to look exactly the same as on Twitch.

**Comment by pajlada** [comment, marked as answer]
This is a tricky responsibility for an application to have, so I'd suggest you look into an application that can do this for you on your OS.
